### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by ajerion** [moralmunky/Home-Assistant-Mail-And-Packages#518] [Q&A] [answered]
I have the repository installed though HACS and can see it in the HACS Integrations.

If I'm understanding the installation instructions correctly, the next step should be go to the main HASS Integration page and add Mail and Packages, but I do not see it in the list.

Am I miss understanding the instructions or have I done something wrong?

Thanks for your help.

**Comment by firstof9** [comment, marked as answer]
Hit `F5` (or whatever the refresh button is if you're using a Mac) in your browser from the `Integrations` page, then hit `+ Add Integration`.

**Reply by ajerion** [reply]
I could have sworn I did that!  Thanks, that worked.
